client/src/components/Header/Header.component.jsx

```diff
@@ -23,10 +28,28 @@ const Header = () => {
   const { user, logout } = useAuth()
 
   const location = useLocation()
+  const { search } = useLocation()
+  const searchParams = new URLSearchParams(search)
   const match = matchPath(location.pathname, {
     path: '/agency/:agency',
   })
-  const agencyShortName = match?.params?.agency
+  const matchPost = matchPath(location.pathname, {
+    path: '/questions/:id',
+  })
+  const postId = matchPost?.params?.id
+  const { data: post } = useQuery([GET_POST_BY_ID_QUERY_KEY, postId], () =>
+    getPostById(postId, 3),
+  )
```

Comment:
Do we really need to bring in related questions? (Hint: the answer may not be as obvious!)

Reply:
updated the PR description with the reasoning behind this approach